src/Domain/CoordinateMaps/Wedge.hpp

```diff
@@ -21,19 +21,49 @@ class er;
 namespace domain {
 namespace CoordinateMaps {
 
+namespace detail {
+// This mapping can be deleted once the 2D and 3D wedges are oriented the same
+// (see issue https://github.com/sxs-collaboration/spectre/issues/2988)
+template <size_t Dim>
+struct WedgeCoordOrientation;
+template <>
+struct WedgeCoordOrientation<2> {
+  static constexpr size_t radial_coord = 0;
+  static constexpr size_t polar_coord = 1;
+  static constexpr size_t azimuth_coord = 2;  // unused
+};
+template <>
+struct WedgeCoordOrientation<3> {
+  static constexpr size_t radial_coord = 2;
+  static constexpr size_t polar_coord = 0;
+  static constexpr size_t azimuth_coord = 1;
+};
+}  // namespace detail
+
 /*!
  * \ingroup CoordinateMapsGroup
  *
- * \brief Three dimensional map from the cube to a wedge.
+ * \brief Map from a square or cube to a wedge.
  * \image html Shell.png "A shell can be constructed out of six wedges."
  *
- * \details The mapping that goes from a reference cube to a three-dimensional
- *  wedge centered on a coordinate axis covering a volume between an inner
- *  surface and outer surface. Each surface can be given a curvature
+ * \details The mapping that goes from a reference cube (in 3D) or square (in
+ *  2D) to a wedge centered on a coordinate axis covering a volume between an
+ *  inner surface and outer surface. Each surface can be given a curvature
  *  between flat (a sphericity of 0) or spherical (a sphericity of 1).
  *
- *  The first two logical coordinates correspond to the two angular coordinates,
- *  and the third to the radial coordinate.
+ *  In 2D, the first logical coordinate corresponds to the radial coordinate,
+ *  and the second logical coordinates correspond to the angular coordinate. In
+ *  3D, the first two logical coordinates correspond to the two angular
+ *  coordinates, and the third to the radial coordinate. This difference
+ *  originates from separate implementations for the 2D and 3D map that were
+ *  merged. The 3D implementation can be changed to use the first logical
+ *  coordinate as radial direction, but this requires propagating the change
+ *  through the rest of the domain code (see issue
+ *  https://github.com/sxs-collaboration/spectre/issues/2988).
```

Comment:
I recall there being some controversy in the past about linking to github from the contents of the codebase. I think the philosophy was that the code itself should be independent of hosting platform.

Reply:
I feel like the usefulness of these links outweighs the coupling to the hosting platform, don't you think?

Reply:
:shrug: yeah, that's reasonable. I'm fine leaving as-is, just wanted to flag it in case anyone has a strong opinion about it.